en/04_Changelogs/5.0.0.md

```diff
@@ -62,6 +63,29 @@ This is a major release and as a result there are a number of breaking API chang
     - `SilverStripe\ORM\Connect\Query`
   - Removed `SilverStripe\ORM\DataList::getGenerator()` (use `getIterator()` instead)
   - Removed the `SilverStripe\ORM\Map_Iterator` class. `SilverStripe\ORM\Map` now uses a generator instead.
+- Removed various API related to `SwiftMailer`
+  - Removed the following classes and interfaces
+    - `SilverStripe\Control\Email\Mailer`
+    - `SwiftMailer`
+    - `SwiftPlugin`
+  - Removed the following methods from `SilverStripe\Control\Email\Email`
+    - Replaced with `Email::getData()`
+      - `BaseURL()`
+      - `IsEmail()`
+    - Has equivalent functionality via `Email::getTextBody()`
+      - `hasPlainPart()`
+      - `findPlainPart()`
+    - Has equivalent functionality via `Email::html()`
+      - `invalidateBody()`
+    - Not replaced with equivalent functionality
+      - `debug()`
+      - `generatePlainPartFromBody()`
+      - `getFailedRecipients()`
+      - `getSwiftMessage()`
+      - `mergeConfiguredEmails()`
+      - `render()`
```

Comment:
This wasn't deprecated in the `4` PR - probably we now need a new PR to deprecate that if it's being removed.

Reply:
It was? https://github.com/silverstripe/silverstripe-framework/blob/4/src/Control/Email/Email.php#L886

Reply:
Note this comment was from 12 days ago -  a lot of changes were made since then. :p
You're correct that _now_ it has been deprecated.